client/my-sites/sidebar-unified/style.scss

```diff
@@ -196,6 +196,46 @@ $font-size: rem( 14px );
 				}
 			}
 		}
+
+		.notice {
+			background-color: #3c434a;
+			/* stylelint-disable-next-line scales/font-weight */
+			font-weight: 300;
+
+			&.is-compact {
+				margin: 0;
+				align-items: center;
+			}
+
+			.notice__icon-wrapper {
+				background-color: transparent;
+				width: 35px;
+
+				.notice__icon-wrapper-drop {
+					top: calc( 50% - 9px );
+					left: calc( 50% - 9px );
+					width: 18px;
+					height: 18px;
+					border-radius: 50%;
+					background: #a7aaad;
```

Comment:
Do we want to introduce a new color here or could this be covered by an existing color variable?

Reply:
Done